src/ebuild.template

```diff
@@ -13,7 +13,6 @@ inherit cargo
 DESCRIPTION="{description}"
 HOMEPAGE="{homepage}"
 SRC_URI="$(cargo_crate_uris ${{CRATES}})"
-RESTRICT="mirror"
```

Comment:
The Gentoo mirrors don't carry the crate source every so this is wasteful to not have.